File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/pub/d/AllergicHandleModeEnum.java
package iih.ci.ord.pub.d;

import xap.sys.appfw.orm.desc.ent.dataobject.DmEnumDesc;

public interface AllergicHandleModeEnum {

    @DmEnumDesc(name="药品禁用",description="由于过敏，药品禁用")
	public static final Integer FORBIDDEN=0;
    @DmEnumDesc(name="再次皮试",description="再次皮试，已测试是否还过敏")
	public static final Integer RESKINTEST=1;
    @DmEnumDesc(name="强制使用",description="情况紧急，必须强制使用")
	public static final Integer FORCEDUSE=2;
}	

File: client/iih_stub/iih.ci_stub/iih.ci.ord/pub/d/AllergicHandleModeEnum.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.ComponentModel;

namespace iih.ci.ord.pub.d
{
   public enum AllergicHandleModeEnum {
	/// <summary>
	/// 药品禁用
	/// </summary>
    [Description("药品禁用")]
    FORBIDDEN=0,
	/// <summary>
	/// 再次皮试
	/// </summary>
    [Description("再次皮试")]
    RESKINTEST=1,
	/// <summary>
	/// 强制使用
	/// </summary>
    [Description("强制使用")]
    FORCEDUSE=2,
   }
}
